apps/server/src/modules/tool/tool-launch/service/launch-strategy/lti11-tool-launch.strategy.ts

```diff
@@ -113,7 +123,7 @@ export class Lti11ToolLaunchStrategy extends AbstractLaunchStrategy {
 					location: PropertyLocation.BODY,
 				})
 			);
-		} else {
+		} else if (config.privacy_permission !== LtiPrivacyPermission.ANONYMOUS) {
 			additionalProperties.push(
 				new PropertyData({
 					name: 'user_id',
```

Comment:
I think in the specification it says not to send userId. If this is not possible, what do you think about at least pseudonymizing the userId ?